src/calculateRentalCost.js

```diff
@@ -20,7 +20,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const countRent = 40;
+  let allPrice = 0;
+
+  if (days > 1) {
+    if (days >= 7) {
+      allPrice = countRent * days - 50;
+    } else if (days >= 3 && days <= 7) {
+      allPrice = countRent * days - 20;
```

Comment:
А, и добавляй кавычки в выражения, чтобы даже задумываться не приходилось о последовательности операторов
```suggestion
      allPrice = (countRent * days) - 20;
```

Reply:
Согласен, сделал